src/runtime/context.rs

```diff
@@ -1,30 +1,27 @@
-use crate::runtime::driver::Driver;
-use crate::util::PhantomUnsendUnsync;
+use crate::runtime::driver;
```

Comment:
Maybe a good time to add a few file comments about this RuntimeContext. How it sort of sits behind the scenes and is ready to potentially give access to the Runtime and gets installed by the Runtime's block_on pub(crate) routine. (I think block_on is currently pub though.)

But whatever you think. You're allowed to say, you don't think it's necessary. I understand sometimes I look for too much help from comments when another window and another minute would answer the question for me.

Reply:
You bring up a good point here, this absolutely needs documented.